Resources/public/js/views/ez-subitemlistview.js

```diff
@@ -43,6 +43,14 @@ YUI.add('ez-subitemlistview', function (Y) {
                 this._uiPageLoading();
                 this._fireLocationSearch();
             });
+
+            this.get('location').on(
+                ['hiddenChange', 'invisibleChange'],
+                Y.bind( function () {
+                    this._uiPageLoading();
```

Comment:
I think you should add a `_refresh` method that calls `_uiPageLoading` and `_fireLocationSearch`, that we can write:
```js
this.after('offsetChange', this._refresh);
this.get('location').after(['hiddenChange', 'invisibleChange'], Y.bind(this._refresh, this));
```

which is bit better in terms of readability